pycbc/io/hdf.py

```diff
@@ -1031,7 +1043,7 @@ def to_coinc_hdf_object(self, file_name):
         # time will be used later to determine active ifos
         ofd['time'] = time
 
-        if self._inclusive:
+        if not exclusive:
```

Comment:
Is the `self_inclusive` flag from before not still relevant or active? Does it need removing in any way?

Reply:
I missed the comment at the top of the function, stating to use has_inc=False for injections, so double-included this